### ISSUE TITLE
[TW-503] "im" in Description replaced through "imask"

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#538]
_Stefan Betz on 2012-07-13T10:39:18Z says:_ 

 Every time i create a task with "im" in description text, the "im" will be replaced by "imask" :-/

<pre>
[stefan(at)pc2007 ~]$ task 34 info

Name        Value                                     
ID          34
Description Funktion vom Zentral-AUS imask HUB prüfen.
Status      Pending
Priority    L
Due         20120716
Tags        baur knx
UUID        be1cb113-fca3-4482-96a5-4cae9e244aec
Entered     20120712 (20 hrs)
Urgency     10.38

[stefan(at)pc2007 ~]$ task 34 mod "Funktion vom Zentral-AUS im HUB prüfen."
Modified 0 tasks.
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-07-13T10:39:18Z
Modified: 2014-02-09T01:55:47Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2012-07-13T12:16:32Z says:_ 

 This is a duplicate of #1024 and has been fixed in the 2.0.1 branch (which is going to be released "soonish").

Thank you for reporting this problem anyway, the more people report issues, the sooner they get fixed :)
